app/notification/model/notification_utils.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+
+def convert_bool_value(data):
+    def convert_values(value):
+        if value is None:
+            return "Not provided"
+        elif value is True:
+            return "Yes"
+        elif value is False:
+            return "No"
+        else:
+            return value
+
+    if isinstance(data, list):
+        if all(isinstance(sublist, list) for sublist in data):
+            converted_data = [
+                [convert_values(value) for value in sublist]
+                for sublist in data
+            ]
+        else:
+            converted_data = [convert_values(value) for value in data]
+    else:
+        converted_data = convert_values(data)
+
+    return converted_data
+
+
+def format_answer(answer):
+    if answer is None:
+        return "Not provided"
+    if "-" in answer:
+        return answer.replace("-", " ")
+    if "null" in answer:
+        return re.sub(r"\s*null\s*,?", "", answer)
+
+    if isinstance(answer, list):
+        return [
+            a.replace("'", "").replace("-", " ")
+            for a in answer
+            if isinstance(a, str)
+        ]
+
+    return answer
+
+
+def simplify_title(section_name, remove_text: list):
+    section = section_name.split("-")
+    simplified_title = []
+
+    for i, text in enumerate(section):
+        if text in remove_text:
+            simplified_title = section[:i]
+            break
+
+    if not simplified_title:
+        simplified_title = section
+
+    return simplified_title
```

Comment:
This could be done with a simple regex, but your way is also valid:

```suggestion
def simplify_title(section_name: str, remove_text: str) -> str:
    pattern = re.compile(f'-{remove_text}(?:-.*|$)')
    prefix = re.sub(pattern, '', section_name)
    return prefix.replace('-', ' ').capitalize()
```

Could we add some unit tests too? Should make support/future changes easier.

```python
@pytest.mark.parametrize(
    "section_name, remove_text, expected_output", [
        ("organisation-information-cof-r3-w1", "cof", "Organisation information"),
        ("operational-costs-cof-r3-w1", "cof", "Operational costs"),
        ("current-services-ns", "ns", "Current services"),
        ("section-without-remove-text", "ns", "Section without remove text"),
        ("section-with-multiple-remove-text-cof-ns", "cof", "Section with multiple remove text"),
        ("section-with-multiple-remove-text-ns-cof", "ns", "Section with multiple remove text"),
        ("unrelatedns-code", "ns", "Unrelatedns code"),
    ]
)
def test_simplify_title(section_name, remove_text, expected_output):
    assert simplify_title(section_name, remove_text) == expected_output
```

Reply:
On second thought, should we even be doing this? I just realised we're re-constructing the section titles from the form name, this data should already be mapped in fund store, I don't think we need to do it all manually,

![image](https://github.com/communitiesuk/funding-service-design-notification/assets/117724519/797e62ef-d180-47ea-ae50-52f2327a2f52)

![image](https://github.com/communitiesuk/funding-service-design-notification/assets/117724519/fe5eaa39-6894-45ee-964c-454dad148b96)

Shouldn't we just be pulling the data from the fund-store?